### PR TITLE
[Docs](mlu-ops):Add mlu-ops BANGC-OPS-OpList.md.

### PR DESCRIPTION
**Pull request by mahxn0**
Thanks for your contribution and we appreciate it a lot. :rocket::rocket:

## 1. Motivation

Add bangc-ops.md.

## 2. Modification

docs/bangc-docs/BANGC-OPS-OpList.md

### CODE REVIEW

File: docs/bangc-docs/BANGC-OPS.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# 算子
```
Comment (DanieeelLiu) [inline docs/bangc-docs/BANGC-OPS.md:1]: 文件名改一下吧，感觉不明确

File: docs/bangc-docs/BANGC-OPS.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+| copy                                   | √         |            | √       |
+| deform_roi_pool_backward               | √         | √          |         |
+| deform_roi_pool_forward                | √         | √          |         |
+| diff_iou_rotated_sort_vertices_forward | √         | √          |         |
+| div                                    | √         | √          |         |
+| dynamic_point_to_voxel_backward        | √         | √          |         |
+| dynamic_point_to_voxel_forward         | √         | √          |         |
+| expand                                 | √         |            | √       |
+| fill                                   | √         |            | √       |
+| fiil_zero                              |           | √          |         |
```
Comment (DanieeelLiu) [inline docs/bangc-docs/BANGC-OPS.md:34]: fiil写错

File: docs/bangc-docs/BANGC-OPS.md (hunk 1 above)
Comment (mahxn0) [reply docs/bangc-docs/BANGC-OPS.md]: 已修改

File: docs/bangc-docs/BANGC-OPS.md (hunk 2 above)
Comment (mahxn0) [reply docs/bangc-docs/BANGC-OPS.md]: 已修改

**Review by DanieeelLiu — APPROVED** [review]

File: docs/bangc-docs/BANGC-OPS-OpList.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
+# 算子
+
+MLU-OPS 提供了检测、分割等任务中常用的算子，支持自定义算子（领域/长尾类算子）．
+
+根据算子在仓库中是否有源码实现分为 MLU Source Op 和 MLU Binary Op．
+
+- MLU Source Op: 源码算子，Device侧代码在仓库有源码实现．
```
Comment (Unireverse) [inline docs/bangc-docs/BANGC-OPS-OpList.md:7]:
```suggestion
- MLU Source Op: 源码算子，Host及Device侧代码在仓库有源码实现。
```

File: docs/bangc-docs/BANGC-OPS-OpList.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+- MLU Binary Op: 二进制算子，Device侧代码在仓库无源码实现，通过调用底层算子库实现.
```
Comment (Unireverse) [inline docs/bangc-docs/BANGC-OPS-OpList.md:9]:
```suggestion
- MLU Binary Op: 非源码算子，Host及Device侧代码在仓库无源码实现，通过调用底层CNNL算子库实现。
```

**Review by duzekunKTH — APPROVED** [review]

File: docs/bangc-docs/BANGC-OPS-OpList.md (hunk 3 above)
Comment (mahxn0) [reply docs/bangc-docs/BANGC-OPS-OpList.md]: 已修改

File: docs/bangc-docs/BANGC-OPS-OpList.md (hunk 4 above)
Comment (mahxn0) [reply docs/bangc-docs/BANGC-OPS-OpList.md]: 已修改

**Review by Unireverse — APPROVED** [review]

**Pull request merged**